README.md

```diff
@@ -47,3 +47,13 @@ above:
 ```bash
     git apply dakota-6.16.patch
 ```
+
+### Installation for Dakota version 6.18 or above
+
+From Dakota version 6.18 a file was removed from the source and build script was altered.
+To support this, apply the following patch to use Carolina with Dakota 6.13, 6.16 and
```

Comment:
`use Carolina with Dakota 6.13, 6.16 and` -> `use Carolina with Dakota 6.18 and`